contestantA/src/main/java/edu/asu/diging/contestantA/web/HomeController.java

```diff
@@ -10,4 +10,19 @@
     public String home() {
         return "home";
     }
+    
+    @RequestMapping(value = "/manual")
+    public String manual() {
+        return "manual";
+    }
+    
+    @RequestMapping(value = "/automatic")
+    public String automatic() {
+        return "automatic";
+    }
+    
+    @RequestMapping(value = "/help")
+    public String help() {
+        return "help";
+    }
```

Comment:
Each new page should have their own controller. So each of these method should be in their own controller.